cloud-sql/sqlserver/mssql/server.js

```diff
@@ -0,0 +1,197 @@
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const express = require('express');
+const mssql = require('mssql')
+const bodyParser = require('body-parser');
+
+const app = express();
+app.set('view engine', 'pug');
+app.enable('trust proxy');
+
+// Automatically parse request body as form data.
+app.use(bodyParser.urlencoded({extended: false}));
+app.use(bodyParser.json());
+
+// Set Content-Type for all responses for these routes.
+app.use((req, res, next) => {
+  res.set('Content-Type', 'text/html');
+  next();
+});
+
+// Create a Winston logger that streams to Stackdriver Logging.
+const winston = require('winston');
+const {LoggingWinston} = require('@google-cloud/logging-winston');
+const loggingWinston = new LoggingWinston();
+const logger = winston.createLogger({
+  level: 'info',
+  transports: [new winston.transports.Console(), loggingWinston],
+});
+
+// [START cloud_sql_server_mssql_create]
+let pool;
+const createPool = async () => {
+  let config = {pool: {}};
+  config.user = process.env.DB_USER; // e.g. 'my-db-user'
+  config.password = process.env.DB_PASS; // e.g. 'my-db-password'
+  config.database = process.env.DB_NAME; // e.g. 'my-database'
+  config.server = process.env.DEPLOYED ? '172.17.0.1' : '127.0.0.1';
+  config.port = 1433;
+
+  // [START_EXCLUDE]
+
+  // [START cloud_sql_server_mssql_timeout]
+  // 'connectionTimeout` is the maximum number of milliseconds to wait trying to establish an
+  // initial connection. After the specified amount of time, an exception will be thrown.
+  config.connectionTimeout = 30000;
+  // 'acquireTimeoutMillis' is the number of milliseconds before a timeout occurs when acquiring a 
+  // connection from the pool.
+  config.pool.acquireTimeoutMillis = 30000;
+  // [END cloud_sql_server_mssql_timeout]
+
+  // [START cloud_sql_server_mssql_limit]
+  // 'max' limits the total number of concurrent connections this pool will keep. Ideal
+  // values for this setting are highly variable on app design, infrastructure, and database.
+  config.pool.max = 5;
+  // 'min' is the minimum number of idle connections maintained in the pool.
+  // Additional connections will be established to meet this value unless the pool is full.
+  config.pool.min = 1;
+  // [END cloud_sql_server_mssql_limit]
+
+  // [START cloud_sql_server_mssql_lifetime]
+  // 'idleTimeoutMillis' is the number of milliseconds a connection must sit idle in the pool 
+  // and not be checked out before it is automatically closed
+  config.pool.idleTimeoutMillis = 600000,
```

Comment:
Idle timeout for the other samples is under "[Connection timeout](https://cloud.google.com/sql/docs/mysql/manage-connections#pools)".

This is different than "Connection duration", which the MySQL sample doesn't have (may need to do something similar here): 
https://cloud.google.com/sql/docs/mysql/manage-connections#duration

Reply:
moved, and also removed the `lifetime` region tag